go/felix/intdataplane/endpoint_mgr.go

```diff
@@ -524,17 +534,23 @@ func (m *endpointManager) resolveHostEndpoints() {
 						// that is on this interface.
 						logCxt.Debug("Match on address")
 						bestHostEpId = id
+						bestHostEp = *hostEp
 						continue HostEpLoop
 					}
 				}
 			}
 		}
 		if bestHostEpId.EndpointId != "" {
-			log.WithFields(log.Fields{
+			logCxt := log.WithFields(log.Fields{
 				"ifaceName":    ifaceName,
 				"bestHostEpId": bestHostEpId,
-			}).Debug("Got HostEp for interface")
+			})
+			logCxt.Debug("Got HostEp for interface")
 			newIfaceNameToHostEpID[ifaceName] = bestHostEpId
+			if len(bestHostEp.UntrackedTiers) > 0 {
+				logCxt.Debug("Endpoint has untracked policies.")
+				newUntrackedIfaceNameToHostEpID[ifaceName] = bestHostEpId
+			}
```

Comment:
No, the default in the tracked policy is to drop or fall through to profiles (and we need to render the failsafe policy too).  Will add a comment.